documentation/src/main/asciidoc/topics/proc_server_credentials.adoc

```diff
@@ -45,13 +46,14 @@ dbpassword
 +
 . Configure {brandname} to use the credential keystore.
 .. Specify the name and location of the credential keystore in the `credential-stores` configuration.
-.. Specify credential keystore and the password alias in the `credential-reference` configuration, if necessary.
+.. Specify the credential keystore and alias in the `credential-reference` configuration, if necessary.
 +
 [TIP]
 ====
-You need to specify the credential keystore in the `credential-reference` configuration only if there is more than one credential keystores.
+The `credential-reference` configuration is necessary only if you have multiple keystores or aliases.
```

Comment:
I think `<credential-reference>` is always required if you want to use the credential store.
What is optional is the attributes:
* store => if you have a single `<credential-store>`.
* alias => if you have a single "alias" inside the keystore.

Reply:
Nicely spotted, Pedro! I've changed it.